src/views/ProductsDetails.vue

```diff
@@ -198,25 +201,57 @@ const submitProduct = async () => {
     productsContent: productsContent.value,
   };
   try {
-    await store.dispatch("products/createProduct", product);
-    return true;
+    const res = await store.dispatch("products/createProduct", product);
+    snackbarProvider.showSuccessSnackbar("Successfully added product!");
+    return res;
   } catch (error) {
     snackbarProvider.showErrorSnackbar("Could not create the product");
   }
   return false;
 };
 
+const handlePictureSelected = (chosenFile) => {
+  selectedPicture.value = chosenFile;
+};
+
+const isPictureValidated = () => {
+  if (selectedPicture.value <= 0) {
+    return false;
+  }
+  return true;
```

Comment:
Does this work in this case?
```suggestion
  return !!selectedPicture.value
```

Reply:
should be the same as the if though
```suggestion
  return selectedPicture.value > 0;
```

Reply:
done